lib/mailserver.js

```diff
@@ -113,47 +113,46 @@ function handleDataStream (stream, session, callback) {
 }
 
 /**
- * Delete everything in the temp folder
+ * Delete everything in the mail directory
  */
-
-function clearTempFolder () {
-  fs.readdir(tempDir, function (err, files) {
+function clearMailDir () {
```

Comment:
This method will always delete at startup the directory so we loose the feature to get back mail if a container restart

Reply:
Yes, I didn't change this, as this behaviour was no issue for me. The persistent volume was needed because of storage restrictions on the container/pod. MailDev was used as a transient (but potentially high volume) testing environment. But you are right that if we already use a persistent volume, it is probably the expected behaviour that this does not get cleared out on restart.